src/main/java/frc/robot/subsystems/ROS.java

```diff
@@ -60,15 +63,17 @@ public static void updateTables() {
     portEncoderEntry.setDouble(Drivetrain.getDriveEncoderPort());
     rosIndex.setNumber(rosIntex);
 
-    // Increase the Index value
+    networkTableInstance.flush(); // Force an update and flush all values out.
+
+    // Increase the Index value (Used for Syncing)
     rosIntex = rosIntex + 1;
     if (rosIntex > 255) {
       rosIntex = 1;
     }
   }
 
-  public double getStarboardSpeed(){return coprocessorStarboard.getDouble(0);}
-  public double getPortSpeed(){return coprocessorPort.getDouble(0);}
+  public double getStarboardSpeed(){return coprocessorStarboard.getDouble(0);} // A number in m/s
+  public double getPortSpeed(){return coprocessorPort.getDouble(0);} // A number in m/s
```

Comment:
is this really in m/s?

Reply:
Yes, or im hoping we design for that, unless you want to use knots again? and translate that on LAPIS
The number coming back here will be in a unit of speed, so our FlyWithWires function can velocity control the wheels